### ISSUE TITLE
Missing semantic tags

### ISSUE THREAD

**Issue by BizTheHabesha** [BizTheHabesha/bisratgebrekidan-module-1-challenge#1]
The HTML is missing semantic tags, making it difficult for certain accessibility features to make proper use of the webpage.
